tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -243,3 +244,56 @@ def test_all_transactions(self):
                 expected_partitions[topic.name]) == topic.partition_count
             for partition in range(topic.partition_count):
                 assert partition in expected_partitions[topic.name]
+
+    @cluster(num_nodes=3)
+    def test_delete_topic_from_tx(self):
```

Comment:
this test looks good, but I think you should use `@matrix` to create more test cases for various scenarios. more transactions, more partitions, more data, etc... if the test is structured in a generic way then you can use `@matrix` to cheaply explore a large state space.